taskcluster/test/params/cron-run-integration-tests.yml

```diff
@@ -26,5 +28,5 @@ pushdate: 0
 pushlog_id: '0'
 repository_type: git
 target_tasks_method: integration
-tasks_for: github-issue-comment
+tasks_for: cron
```

Comment:
Are we getting rid of the on-demand integration tests? If not, maybe this file ought to be copied instead of moved? (Or maybe this cron version of the file is a superset, so it would provide all the coverage needed...?)